src/compiler/commandLineParser.ts

```diff
@@ -294,6 +294,10 @@ namespace ts {
             name: "allowJs",
             type: "boolean",
             description: Diagnostics.Allow_javascript_files_to_be_compiled
+        },
+        {
+            name: "listEmit",

```

Comment:
Can you steal a name for this parameter from another compiler? I think there's probably an established convention out there.